packages/web-scripts/src/integration.test.ts

```diff
@@ -200,7 +200,7 @@ describe('integration tests', () => {
 
   async function testScripts(
     buildArgs: string[] = [],
-    lintArgs: string[] = [],
+    lintArgs: string[] = ['--ignore-path=.gitignore', '--format=checkstyle'],
```

Comment:
Added reproduction from #341 as a failing test, though I guess it's a bit random to be testing with these specific options set. Perhaps better broken out as a separate test.

Reply:
I think this is good for now - we have a separate backlog task to break this IT up into something more manageable for all of the scripts.